plugins/inputs/socket_listener/stream_listener.go

```diff
@@ -83,6 +86,31 @@ func (l *streamListener) setupUnix(u *url.URL, tlsCfg *tls.Config, socketMode st
 	return nil
 }
 
+func (l *streamListener) setupVsock(u *url.URL) error {
+	var err error
+
+	addrTuple := strings.SplitN(u.String(), ":", 2)
+
+	//check if address string has two tokens
```

Comment:
Please leave a space between `//` and the comment as otherwise this might be interpreted as a comment targetted for a machine/go directive...
```suggestion
	// Check address string for containing two tokens
```